dist/s_funcs.list

```diff
@@ -53,3 +53,7 @@ wiredtiger_unpack_item
 wiredtiger_unpack_start
 wiredtiger_unpack_str
 wiredtiger_unpack_uint
+__wt_page_npos
+__wt_page_from_npos
+__wt_page_from_npos_for_eviction
```

Comment:
__wt_page_from_npos_for_eviction and __wt_page_npos should be removable from this list.

Reply:
This is currently the way to make functions externally visible.

Reply:
This list is functions that don't have an obvious external caller. To make functions externally visible you add them to extern.h. Those two functions can be removed.